src/kernels/Conv_Winograd_v30_3_1_gfx9_fp32_f3x2_stride2.inc

```diff
@@ -421,47 +461,47 @@ v_xor_b32_e32 v92, 0x30f, v105
 v_mad_u32_u24 v92, 4, v92, v107
 v_xor_b32_e32 v93, 8, v105
 v_mad_u32_u24 v93, 4, v93, v107
-v_subrev_co_u32_e32 v95, vcc, s56, v95
-v_mov_b32_e32 v105, s55
+v_subrev_co_u32_e32 v95, vcc, s58, v95
+v_mov_b32_e32 v105, s57
 v_cmp_lt_i32_e32 vcc, v95, v105
 v_subb_co_u32_e64 v104, vcc, 0, 0, vcc
-v_mad_i32_i24 v95, v104, s55, v95
-v_mad_i32_i24 v97, v104, s60, v97
-v_mad_i32_i24 v96, v104, s59, v96
-v_mov_b32_e32 v105, s54
+v_mad_i32_i24 v95, v104, s57, v95
+v_mad_i32_i24 v97, v104, s62, v97
+v_mad_i32_i24 v96, v104, s61, v96
+v_mov_b32_e32 v105, s56
 v_cmp_lt_i32_e32 vcc, v96, v105
 v_subb_co_u32_e64 v104, vcc, 0, 0, vcc
 v_add_co_u32_e32 v97, vcc, v97, v104
 v_mad_i32_i24 v96, v104, v105, v96
-v_subrev_co_u32_e32 v96, vcc, s57, v96
+v_subrev_co_u32_e32 v96, vcc, s59, v96
 v_cmp_lt_i32_e32 vcc, v96, v105
 v_subb_co_u32_e64 v104, vcc, 0, 0, vcc
 v_add_co_u32_e32 v97, vcc, v97, v104
-v_mad_i32_i24 v96, v104, s54, v96
-v_subrev_co_u32_e32 v97, vcc, s58, v97
-s_mov_b32 s11, 0
-s_mov_b32 s38, s28
+v_mad_i32_i24 v96, v104, s56, v96
+v_subrev_co_u32_e32 v97, vcc, s60, v97
+s_mov_b32 s35, 0
+s_mov_b32 s38, s24
 s_mov_b32 s39, 1
-s_mov_b32 s64, 0
-s_mov_b32 s65, s16
-s_mov_b32 s63, s65
-s_sub_u32 s72, -1, s71
-s_sub_u32 s72, s72, 32
-s_bitset1_b32 s18, 21
-s_mov_b32 s83, 0
+s_mov_b32 s66, 0
+s_mov_b32 s67, s12
+s_mov_b32 s65, s67
+s_sub_u32 s75, -1, s74
+s_sub_u32 s75, s75, 32
+s_bitset1_b32 s14, 21
 s_mov_b32 s87, 0
+s_mov_b32 s91, 0
 v_add_co_u32_e32 v104, vcc, 2, v0
 v_bfe_u32 v104, v104, 2, 1
 v_cmp_ne_u32_e64 vcc, v104, 1
-s_mov_b64 s[2:3], vcc
-s_mov_b32 s73, 17
-s_mov_b32 s62, 0
-s_bitset1_b32 s18, 26
-s_call_b64 s[4:5], 1589
+s_mov_b64 s[6:7], vcc
+s_mov_b32 s76, 17
+s_mov_b32 s64, 0
+s_bitset1_b32 s14, 26
+s_call_b64 s[36:37], 1589
 v_cmp_le_u32_e32 vcc, 0x100, v0
 s_cbranch_vccnz 1
 s_branch 815
-s_mov_b64 vcc, s[2:3]
+s_mov_b64 vcc, s[6:7]
 v_mac_f32_e32 v4, v38, v46
```

Comment:
gfx94x ASICs don't support `v_mac_f32` instruction, use `v_fmac_f32` instead (see [gfx940 ISA](https://llvm.org/docs/AMDGPU/AMDGPUAsmGFX940.html)).
Please check other `Conv_Winograd_v30_3_1_gfx9_*` sources for the same error.

Other possible unsupported instructions:
`v_mac_f32` --> `v_fmac_f32`
`v_mad_f32` --> `v_fma_f32`
`v_madak_f32` --> `v_fmaak_f32`
`v_madmk_f32` --> `v_fmamk_f32`

Reply:
I've got updated version for gfx94x and going to update this PR.